app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
```

Comment:
You can also add validations

Reply:
Yes, of course, still didn't get there :)